app/models/miq_request_workflow.rb

```diff
@@ -1022,7 +1022,7 @@ def vm_or_template_to_hash_struct(ci)
   end
 
   def ems_folder_to_hash_struct(ci)
-    build_ci_hash_struct(ci, [:name, :is_datacenter?])
+    build_ci_hash_struct(ci, [:name, :type])
```

Comment:
@gmcculloug instead of is_datacenter? true/false type will be "EmsFolder" or "Datacenter"
I tested provisioning a VM from a template and the Datacenter and Folder lists look right, is there anything else I should test to confirm?